alf/utils/tensorrt_utils.py

```diff
@@ -389,23 +555,22 @@ def tensorrtify_method(module, method_name):
     Args:
         module: a torch.nn.Module
         method_name: the method name of the module
+        engine_class: should be either ``TensorRTEngine`` or ``OnnxRuntimeEngine``
     """
-    assert is_available(), 'ONNX/TensorRT is not installed!'
-
-    global _tensorrtified_methods
+    global _compiled_methods
     key = (module, method_name)
-    # Here we check if a previous ``tensorrtify_method`` has already been called
-    # for the same module and method. We do not allow tensorrtifying a method
+    # Here we check if a previous ``compile_method`` has already been called
+    # for the same module and method. We do not allow compiling a method
     # a multiple times.
-    assert key not in _tensorrtified_methods, (
-        f"Method {module}.{method_name} is already tensorrtified: "
-        f"{_tensorrtified_methods[key]}")
+    assert key not in _compiled_methods, (
+        f"Method {module}.{method_name} is already compiled: "
```

Comment:
It may be compiled using a different engine_class?

Reply:
> It may be compiled using a different engine_class?

Currently I don't see any necessity of doing this? This assumption simplifies the check.